File: cfgs/java_cfgs_foo/CHECK_IF_A_NUMBER_IS_POWER_OF_ANOTHER_NUMBER.java
Nodes
1;special;;;
2;conditional;foo_1 == 1;14;
3;statement;return foo_1 == 1;14;
4;exit;;;
5;statement;int pow = 1;15;
6;conditional;foo_1 < foo_2;16;
7;statement;pow = foo_1 * foo_2;16;
8;statement;return foo_1 == foo_2;17;
Edges
0;2;
1;3;5;
2;4;
3;
4;6;
5;8;7;
6;6;
7;4;

File: cfgs/pythoncfgs/CHECK_IF_A_NUMBER_IS_POWER_OF_ANOTHER_NUMBER.py
Nodes
1;conditional;foo_1 == 1;8;
2;statement;foo_1 = 1;10;
3;conditional;foo_1 < foo_2;11;
4;statement;return foo_1 == foo_2;13;
5;statement;foo_1 = foo_2 * foo_3;12;
6;statement;return foo_1 == 1;9;
7;exit;;
Edges
0;2;6;
1;3;
2;4;5;
3;7;
4;3;
5;7;
6;
